var/spack/repos/builtin/packages/tau/unwind.patch

```diff
@@ -0,0 +1,32 @@
+From 5206c455e4b77c650614c1a92a4bbfe2ba8e9094 Mon Sep 17 00:00:00 2001
+From: ci <ci>
+Date: Wed, 3 Jun 2020 14:44:17 -0700
+Subject: [PATCH] check unwind_dir for lib or lib64
+
+---
+ configure | 9 ++++++++-
+ 1 file changed, 8 insertions(+), 1 deletion(-)
+
+diff --git a/configure b/configure
+index 3bd7430..0872f24 100755
+--- a/configure
++++ b/configure
+@@ -1663,7 +1663,14 @@ for arg in "$@"; do
+ 	    unwind_dir=`echo $arg | sed -e 's/-unwind=//'`
+         if [ $unwind_dir != "/usr" ]; then
+ 	      unwind_inc="$unwind_dir/include"
+-	      unwind_lib="$unwind_dir/lib"
++	      unwind_lib=""
++	      if [[ -d "$unwind_dir/lib" ]] ; then
++                unwind_lib="$unwind_dir/lib"
++              elif [[ -d "$unwind_dir/lib64" ]] ; then
```

Comment:
The indentation here looks a little weird

Reply:
The indentation is funky all over the Tau configure file, without the patch. I'm fixing this portion, but it is what it is.